.github/workflows/build_and_deploy.yml

```diff
@@ -19,6 +19,11 @@ jobs:
         run: |
           npm ci
           npm run build-storybook
+      - name: Preview Publish 🦫
+        env:
+          GITHUB_TOKEN: ${{ secrets.MARSHMALLOW_CI_PAT }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release --dry-run
```

Comment:
Just to understand the value for this - if there is an issue that's found in the dry run, do we know if the workflow would quit on this step or carry on?